docs/Bundle-Lifecycle.md

```diff
@@ -0,0 +1,48 @@
+## Lifecycle of a Bundle
+
+<img src='./images/codalab-bundle-states.png' />
```

Comment:
This figure can be improved...what you had in the demo was actually nicer...  And this should be separated per bundle type.